### DISCUSSION TITLE
Rack Device Placement: Side by Side in a Rack Unit (U)?

### DISCUSSION THREAD

**Discussion by JessicaRigol** [netbox-community/netbox#16663] [Q&A] [answered]
I'm practicing documentation and would like to know if it's possible to place the devices side by side instead of one below the other within the same rack unit (U). 

For example, as it is: 
![image](https://github.com/netbox-community/netbox/assets/112140945/2b982f16-5d60-4d8a-94df-65d93c8e4319)

I would prefer them side by side. Is that possible?

Thanks!

**Comment by a084ed22** [comment, marked as answer]
No, this is not possible.

**Comment by MattDRogers** [comment]
The official [NetBox docs suggest](https://github.com/netbox-community/netbox/wiki/Frequently-Asked-Questions#how-can-i-model-half-width-devices) creating a Parent device and adding the two devices that occupy the single RU as Child devices. It's not perfect, but it works, though it does introduce a layer of abstraction that I do not prefer.

In my situation, I put the two device in the same RU, but the device on the left side is assigned to the front of the rack, while the device on the right is assigned to the rear of the rack. No Parent/Child stuff. All modeled half-width devices cannot be full-depth for this to work. Again, not perfect, but it's just how we do it.

I hope you find a good method that works for you. If you come up with something cool, let us know.

**Comment by Vicentola** [comment]
@JessicaRigol Can you send me a screenshot of the configuration you made to apply to my hack too?
